packages/containers/src/DeleteResource/sagas.js

```diff
@@ -23,8 +23,7 @@ got ${resourcePath}`,
 	return resourceType;
 }
 
-export function* redirect(action) {
-	const url = get(action, 'data.model.redirectUrl');
+export function* redirect(url) {
 	if (!url) {
 		throw new Error('redirect action must have data.model.redirectUrl value');
```

Comment:
This message should be updated since we're no longer testing the `redirectUrl` object's key